worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageMappingCallable.java

```diff
@@ -293,11 +302,11 @@ private IBaseBundle getBundle(String contractNumber, int searchMonth, int search
      * @return if we're in sandbox, return the synthetic data year unless it's the new Synthea data which can use
      * the correct year
      */
-    int getCorrectedYear(Contract contract, int coverageYear) {
+    int getCorrectedYear(Ab2dEnvironment appEnv, int coverageYear) {
 
         // Synthea contracts use realistic enrollment reference years so only original
         // synthetic contracts need to have the year modified
-        if (contract.getContractType() == Contract.ContractType.CLASSIC_TEST) {
+        if (appEnv == Ab2dEnvironment.SANDBOX) {
```

Comment:
This is not right at all @sharshar-sb used the contract type for a specific reason. Some test contracts need the synthetic year others do not.